packages/core/content-manager/admin/src/preview/components/PreviewHeader.tsx

```diff
@@ -57,6 +57,7 @@ const ClosePreviewButton = () => {
 
   return (
     <IconButton
+      style={{ border: 'none' }}
```

Comment:
I thought it was overkill to created a styled component for this, but happy to do it if y'all think it's better

Reply:
I don't mind a quick style tag here and there. But have you tried variant="ghost"?

Reply:
Yup good call thanks! That's me just being a dumb dumb